ansible/roles/containerd/tasks/install.yml

```diff
@@ -0,0 +1,77 @@
+---
+- name: Copy containerd containerd.conf file
+  template:
+    src: "containerd.conf.j2"
+    dest: "/etc/modules-load.d/containerd.conf"
+  become: yes
+
+- name: "modprobe overlay"
+  shell: "modprobe overlay"
+  become: yes
+
+- name: "modprobe br_netfilter"
+  shell: "modprobe br_netfilter"
+  become: yes
+
+- name: Ensure no old docker version is installed
+  yum:
+    name: "{{ docker_old_packages }}"
+    state: absent
+  become: yes
+
+- name: Install dependencies needed by containerd
+  package:
+    name: "{{ containerd_dependencies }}"
+    state: present
+  become: yes
+
+- name: Add docker repository
+  yum_repository:
+    name: docker
+    description: Docker
+    baseurl: https://download.docker.com/linux/centos/7/$basearch/stable
+    enabled: 1
+    gpgcheck: 1
+    gpgkey: https://download.docker.com/linux/centos/gpg
+  become: yes
```

Comment:
Does it make sense to include within this `install` file, tasks concerning docker with containerd. I think that it should be better to move the docker stuffs into a separate `tasks` file. WDYT ?

Reply:
This role contains only containerd installation steps. Containerd is provided from the docker repository.

Reply:
I'll check to move the docker installation removal step as a call to the Docker role.

Reply:
Moved the removal of previous docker versions to an independent task on the docker role and added a call to it both on the docker installation task as well as in the containerd installation task.